google/resource_compute_region_instance_group_manager.go

```diff
@@ -346,6 +395,63 @@ func resourceComputeRegionInstanceGroupManagerRead(d *schema.ResourceData, meta
 	return nil
 }
 
+// Updates an instance group manager by applying an update strategy (REPLACE, RESTART) respecting a rolling update policy (availability settings,
```

Comment:
This long parenthetical doesn't read very fluently - can you take another look at this comment?

Reply:
I updated the comment, is it better now ?